plugins/scaffolder/src/components/TemplateCard/TemplateCard.tsx

```diff
@@ -48,6 +53,7 @@ export type TemplateCardProps = {
   title: string;
   type: string;
   name: string;
+  entityTemplate: Entity;
```

Comment:
if we are going to have a prop like this, it should probably be just `template`, and then remove title type and name so that it decides those itself, right?